pyproject.toml

```diff
@@ -46,6 +46,20 @@ module = [
 ]
 ignore_errors = true
 
+[[tool.mypy.overrides]]
+module = [
+    'onnx.npx.npx_core_api',
+    'onnx.npx.npx_graph_builder',
+    'onnx.npx.npx_functions',
+    'onnx.npx.npx_functions_test',
+    'onnx.npx.npx_jit_eager',
+    'onnx.npx.npx_tensors',
+    'onnx.npx.npx_types',
+    'onnx.npx.npx_var',
+    'onnx.test.npx_test',
+]
+ignore_errors = true
```

Comment:
I would use ignores inside files. It also seems dangerous to ignore all errors (which turns off mypy completely)?